docs/csharp/programming-guide/classes-and-structs/snippets/destructors/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿//<Snippet1>
+class First
+{
+    ~First()
+    {
+        System.Diagnostics.Trace.WriteLine("First's destructor is called.");
```

Comment:
We've been standardizing on "finalizer". As long as we're making these changes, let's make that one as well:

```suggestion
        System.Diagnostics.Trace.WriteLine("First's finalizer is called.");
```